### PR TITLE
[Behat] Notifcation context extraction

### PR DESCRIPTION
**Pull request by Arminek**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| License         | MIT

### CODE REVIEW

File: etc/behat/services/contexts/ui.xml [hunk 1]
```diff
@@ -27,6 +27,7 @@
         <parameter key="sylius.behat.context.ui.admin.managing_taxons.class">Sylius\Behat\Context\Ui\Admin\ManagingTaxonsContext</parameter>
         <parameter key="sylius.behat.context.ui.admin.managing_tax_rate.class">Sylius\Behat\Context\Ui\Admin\ManagingTaxRateContext</parameter>
         <parameter key="sylius.behat.context.ui.admin.managing_zones.class">Sylius\Behat\Context\Ui\Admin\ManagingZonesContext</parameter>
+        <parameter key="sylius.behat.context.ui.admin.notification.class">Sylius\Behat\Context\Ui\Admin\NotificationContext</parameter>
```
Comment (lchrusciel) [inline etc/behat/services/contexts/ui.xml]: Can we use it only on admin side?

File: src/Sylius/Behat/Context/Ui/Admin/NotificationContext.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    {
+        $this->notificationChecker = $notificationChecker;
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully created
+     */
+    public function iShouldBeNotifiedItHasBeenSuccessfullyCreated()
+    {
+        $this->notificationChecker->checkNotification('has been successfully created.', NotificationType::success());
```
Comment (lchrusciel) [inline src/Sylius/Behat/Context/Ui/Admin/NotificationContext.php]: Is there any benefit in checking 'has been'? Or even `has been successfully`? We assert a notification type and a key word(_created/updated/deleted_) anyway.

File: etc/behat/services/contexts/ui.xml (hunk 1 above)
Comment (Arminek) [inline etc/behat/services/contexts/ui.xml]: Right now we use it only in managing contexts so it is related with admin side. I think we won't have different use case.

Comment (lchrusciel) [inline etc/behat/services/contexts/ui.xml]: Just wonder if there are any crud notification on show pages. But we can move it later.

Comment (michalmarcinkowski) [inline etc/behat/services/contexts/ui.xml]: That is correct for now :)

File: src/Sylius/Behat/Context/Ui/Admin/NotificationContext.php (hunk 2 above)
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Context/Ui/Admin/NotificationContext.php]: I would leave it as it is now. There is one edge case when we could benefit from that, there could be a notification `has not been successfully created.` 💥  :trollface:

**Comment by pjedrzejewski** [comment]
Awesome, thanks Arek!

**Pull request merged**